### ISSUE TITLE
Shorthand assignment operators

### ISSUE THREAD

**Issue by akhilmd** [Rahul-RB/PyRust#4]
RustLexer.py has the following:
```python
t_TIMESEQUAL        = r'\*='
t_DIVEQUAL          = r'/='
t_MODULUSEQUAL      = r'%='
t_PLUSEQUAL         = r'\+='
t_MINUSEQUAL        = r'-='
```
For this, implement required production rules.

**Comment by Rahul-RB** [comment]
Sure, convert them into functions and return the token taken as input.

**Comment by akhilmd** [comment]
@Rahul-RB, no, we'll need to add another production rule to replace `EQUALS` in the `assignStmt` production rule and alter the `rhs` variable in its action by adding a `BinaryOp` node based on the given shorthand operator.

But, this is not priority and may break assignments, so we can do it later.

**Comment by ganesh-k13** [comment]
Is this still a thing? I want to do it.

**Comment by akhilmd** [comment]
Well, there are a lot critical WIP changes [here](https://github.com/akhilmd/PyRust/tree/non-ast-opt). I think we should wrap that up before trying this. We should implement this only if it does not cause any adverse behaviour in the optimisations part.
